frontend/workflows/serverexperimentation/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@clutch-sh/server-experimentation",
+  "version": "0.0.0-beta",
```

Comment:
```suggestion
  "version": "0.0.0-beta",
  "private": true,
```

Reply:
Change commited and the PR is still green 🙂